### DISCUSSION TITLE
Installing Tekton Operator With Configuration Changes

### DISCUSSION THREAD

**Discussion by lihihason** [tektoncd/operator#435] [Q&A]
Hi,

I am trying to deploy the Tekton Operator (profile: all) on EKS kubernetes cluster.

1. Does the operator support creating Ingress for the dashboard (and change the service type to NodePort instead of clusterIp)?
2. In case i want to use "Import Tekton Resources" from private github repository, i should create secret & service account & rbac objects my own or the operator support it? 
3. Are there any installations  examples documented with configuration changes i could look for?

Thank you 🙏

**Comment by vdemeester** [comment]
Hey @lihihason,

1. It does not support creating ingress for the dashboard as far as I know. This would be a nice feature request though (We should create an [new feature request](https://github.com/tektoncd/operator/issues/new?assignees=&labels=kind%2Ffeature&template=feature-request.md) for this).
2. You should create them on your own. The Operator's job is to install tekton components, rest is kind-of up to you, "import Tekton Resource" is too for example. 
3. There is a relatively lack of documentation on the Operator, and we need to fix this and provide examples for sure 👍🏼 . There is a few documentation [here](https://github.com/tektoncd/operator/tree/main/docs) but it is far from being complete.

Cheers

**Reply by lihihason** [reply]
thanks @vdemeester for your reply.
1. i've created a [feature request](https://github.com/tektoncd/operator/issues/439#issue-1019758237).
meanwhile, do i have any workaround for this? if i'll create the ingress myself, do i have an option to tell the operator to change the service type from clusterIP to NodePort?
2. 👍🏼 thank you
3. ok thank you, adding examples will be good idea!

great day!

**Reply by vdemeester** [reply]
That's a good question on 1., I am not sure so I'll invoke @nikhil-thomas and @AlanGreene
